source/localizable/getting-started/quick-start.md

```diff
@@ -211,13 +211,6 @@ curly braces (`{{component}}`). We're going to tell our component:
    provide this route's `model` as the list of people.
 
 ```app/templates/scientists.hbs{-1,-2,-3,-4,-5,-6,-7,+8}
```

Comment:
@acorncom yep, I ended tracking exactly that commit which is wrong as you figured.
I'd let others with better styling skills do the job but just doing something easy, what about ...

<img width="749" alt="diff" src="https://cloud.githubusercontent.com/assets/52642/13201295/fae05cd8-d848-11e5-8cb4-635488c666de.png">